### ISSUE TITLE
The latest version of SB does not support Chinese paths.

### ISSUE THREAD

**Issue by XMDS** [sannybuilder/dev#102]
@x87  Maybe my expression is not clear. Because I use Chinese, I don't know whether the system in other languages supports it. However, the new version of SB can only exist in English paths. When my folder name has Chinese characters, it will prompt that SCM.ini and OP.txt cannot be found (the old version does not have this problem)

**Comment by x87** [comment]
this sounds similar to #89 

@XMDS Can you track down to what is the last version of SB that supported non-English characters?

**Comment by XMDS** [comment]
> this sounds similar to #89
> 
> @XMDS Can you track down to what is the last version of SB that supported non-English characters?

yes, I just tested the previous version, and this problem has appeared since 3.6.0

**Comment by x87** [comment]
I'll investigate more on this. Meanwhile a workaround is to install Sanny Builder in a directory with only English characters in the path.

**Comment by x87** [comment]
If there are non-English characters in the Sanny path, the compiler fails with "Unknown operator" error (ERR 0098)
